### DISCUSSION TITLE
快照恢复提示成功，但网站和容器还有应用一个没恢复

### DISCUSSION THREAD

**Discussion by xyxmos** [1Panel-dev/1Panel#6371] [Q&A]
### 1Panel 版本

1.10.14

### 问题描述

[BUG]快照恢复提示成功，但网站和容器还有应用一个没恢复

### 重现步骤

先备份快照，在另一台服务器恢复

**Comment by ssongliu** [comment]
感谢反馈，麻烦提供一下系统日志
